src/marten/db/query/sql/raw_statement_node.cr

```diff
@@ -0,0 +1,99 @@
+module Marten
+  module DB
+    module Query
+      module SQL
+        class RawStatementNode < PredicateNode
+          getter statement
+          getter params
+
+          def initialize(
+            @statement : String,
+            @params = [] of ::DB::Any,
+            @children = [] of PredicateNode,
+            @connector = SQL::PredicateConnector::AND, @negated = false, *args
```

Comment:
Will this class be ever instantiated from a statement/params and additional `args`? It doesn't appear to be the case. I guess we could remove this argument altogether. What do you think?